likecoin/assets/js/likecoin.js

```diff
@@ -67,20 +67,34 @@ async function handleUpdateId(newId, newWallet) {
   updateStatus.textContent = await res.text();
 }
 
+async function fetchLikeCoinID(currentAddress) {
+  show('.loading');
+  const res = await fetch(`${CHALLENGE_URL}?wallet=${currentAddress}`);
+  hide('.loading');
+  address = currentAddress;
+  if (res.status === 404) {
```

Comment:
Also, any try-catch block needed?

Reply:
I think every 4xx resolves, but we probably dont have proper way to handle other cases.
Since 404 is throwing error, probably just let all error throw

Reply:
anyway, going to replace this with jquery